### PR TITLE
ARCHI-217: add cluster id on ClusterManager interface

### PR DESCRIPTION
**Pull request by guillaumelamirand**
**Issue**

https://gravitee.atlassian.net/browse/ARCHI-217

**Description**

Add cluster id on ClusterManager

**Additional context**

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `3.1.0-archi-217-distributed-sync-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/node/gravitee-node/3.1.0-archi-217-distributed-sync-SNAPSHOT/gravitee-node-3.1.0-archi-217-distributed-sync-SNAPSHOT.zip)

### CODE REVIEW

File: gravitee-node-api/src/main/java/io/gravitee/node/api/cluster/MemberListener.java [hunk 1]
```diff
@@ -23,6 +23,4 @@ public interface MemberListener {
     void onMemberAdded(final Member member);
 
     void onMemberRemoved(final Member member);
-
-    void onMemberChanged(final Member member);
```
Comment (jhaeyaert) [inline gravitee-node-api/src/main/java/io/gravitee/node/api/cluster/MemberListener.java:27]: Can you remember me what are the reasons behind the removal of `onMemberChanged` method?

**Review by jhaeyaert — APPROVED** [review]
Looks good. I've just added a question for my own understanding.

File: gravitee-node-api/src/main/java/io/gravitee/node/api/cluster/MemberListener.java (hunk 1 above)
Comment (guillaumelamirand) [reply gravitee-node-api/src/main/java/io/gravitee/node/api/cluster/MemberListener.java:27]: There is actually no way to detect a member changed...so this method wasn't used.

**Pull request merged**